test/unit/utilization/docker-info.test.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const fs = require('node:fs')
+const http = require('node:http')
+const os = require('node:os')
+const helper = require('../../lib/agent_helper')
+const standardResponse = require('./aws-ecs-api-response.json')
+const { getBootId } = require('../../../lib/utilization/docker-info')
+
+tap.beforeEach(async (t) => {
+  t.context.orig = {
+    fs_access: fs.access,
+    os_platform: os.platform
+  }
+  fs.access = (file, mode, cb) => {
```

Comment:
this is purely informational but a lot of our tests use sinon for mocking.

`sinon.stub(fs, 'access').yields(Error('no proc file'))`